### PR TITLE
merge to master PR

### PR DESCRIPTION
**Pull request by ansaryergesh**

### CODE REVIEW

File: enumerable_method.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
+module Enumerable
+
+    def my_each 
+      i=0
```
Comment (bellom) [inline enumerable_method.rb]: Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [this link](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide).

File: enumerable_method.rb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      while i<self.length
+       yield(self[i])
+       i+=1
```
Comment (bellom) [inline enumerable_method.rb]: Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [this link](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide).

File: enumerable_method.rb [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      end
+      self
+    end
+    
+    def my_each_with_index
+      i=0
```
Comment (bellom) [inline enumerable_method.rb]: Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [this link](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide).

File: enumerable_method.rb [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+      while i<self.length
+        yield(self[i], i )
+        i+=1
```
Comment (bellom) [inline enumerable_method.rb]: Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [this link](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide).

File: enumerable_method.rb [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        array.push(yield(x))
+      end
+      array
+  
+    end
+  
+    def my_inject (*item)
+      val = self[0]
+      for i in self[1..self.length] do
+        val=yield(val,i)
```
Comment (bellom) [inline enumerable_method.rb]: Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [this link](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide).

**Review by bellom — CHANGES_REQUESTED** [review]
Hi Team,
Well done guys, kindly be aware that you are to use `Git Flow` but instead you used `Github`, make sure to use 'Git Flow' in the next project - `OOP`
Short Explanation Hint: 
`GitHub Flow`: Pull Requests is raise from feature branches to the master.
`Git Flow`: Pull Requests is raise from feature branches to the development.
**Two things to do to;**
1. Add linter's check and fix all errors if need too, checkout this [linters-config#troubleshooting](https://github.com/microverseinc/linters-config#troubleshooting) for help. Example below;
image
If you are not having this check on your Github page, kindly visit :point_right:[ linter set up](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml)
2. You need to include space around operators and indentation with two spaces should be applied.

_Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you_

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
Hello @ansaryergesh ✋ 

Nice work @ansaryergesh but you have to fix those linter errors to get 👍 .we can not approve project with linter errors because it looks unprofessional.

Please submit for anther review after making those changes
Happy Coding

**Review by bellom — APPROVED** [review]
Hi Team, 
**Make sure to use 'Git Flow' in the next project - `OOP`**
Short Explanation Hint: 
`GitHub Flow`: Pull Requests is raise from feature branches to the master.
`Git Flow`: Pull Requests is raise from feature branches to the development.

#### Congratulation :+1: 
Go head and merge !

**Pull request merged**